x-pack/plugins/case/server/routes/api/get_case.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { schema } from '@kbn/config-schema';
+import { RouteDeps } from '.';
+import { wrapError } from './utils';
+
+export function initGetCaseApi({ caseService, router }: RouteDeps) {
+  router.get(
+    {
+      path: '/api/cases/{id}',
+      validate: {
+        params: schema.object({
+          id: schema.string(),
+        }),
+      },
+    },
+    async (context, request, response) => {
+      let theCase;
+      try {
+        theCase = await caseService.getCase({
+          client: context.core.savedObjects.client,
+          caseId: request.params.id,
+        });
+      } catch (error) {
+        return response.customError(wrapError(error));
+      }
+      try {
+        const theComments = await caseService.getAllCaseComments({
+          client: context.core.savedObjects.client,
+          caseId: request.params.id,
+        });
+        return response.ok({ body: { case: theCase, comments: theComments } });
```

Comment:
@XavierM for the `getCase` request, I decided to handle the additional request to return the comments for the case as well. Should I do that, or should we leave that up to the API consumer?

Reply:
I think that will add an options on the `getCase` API to include or not the comments and still create a specific API `getComments` by case id.